### PR TITLE
Fix batch transactions fallbacks

### PR DESCRIPTION
**Pull request by CiprianDraghici**
### Issue/Feature
Redundant fallback actions resulting in to many requests
### Reproduce
Issue exists on version `2.13` of sdk-dapp.

### Contains breaking changes
[x] No

[] Yes

### Updated CHANGELOG
[x] Yes

### Testing
[x] User testing
[] Unit tests

### CODE REVIEW

**Review by MiroMargineanu — APPROVED** [review]

File: src/hooks/transactions/helpers/isOlderThan.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export const isOlderThan = (batchId: string, olderThanMs: number) => {
```
Comment (arhtudormorar) [inline src/hooks/transactions/helpers/isOlderThan.ts:1]: add a comment saying what does this do
I would not send a batchId here. I would send 2 numbers, and call it `timestampIsOlderThan`

File: src/hooks/transactions/helpers/extractSessionId.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+export const extractSessionId = (batchId: string) => {
+  const timestamp = batchId.split('-')[0];
+
+  if (/^\d+$/.test(timestamp)) {
```
Comment (arhtudormorar) [inline src/hooks/transactions/helpers/extractSessionId.ts:4]: isValidNumber

File: src/hooks/transactions/tests/extractSessionId.test.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  test('should return null for an invalid batchId', () => {
+    const batchId = 'invalid-batch-id';
+
+    const result = extractSessionId(batchId);
+
+    expect(result).toBeNull();
+  });
+
+  test('should return null for an empty batchId', () => {
+    const batchId = '';
```
Comment (arhtudormorar) [inline src/hooks/transactions/tests/extractSessionId.test.ts:21]: add test for undefined and null

**Review by arhtudormorar — APPROVED** [review]
